src/main/groovy/co/tala/performance/flo/FloRunnaSettings.groovy

```diff
@@ -47,13 +51,29 @@ class FloRunnaSettings {
         }
 
         int defaultThreads = 8
+        int defaultIterations = 0
         long defaultDuration = 8000
         long defaultRampup = 1000
 
         this.threads = getPropValue("threads", defaultThreads).toInteger()
+        this.iterations = getPropValue("iterations", defaultIterations).toInteger()
         this.duration = getPropValue("duration", defaultDuration).toLong()
         this.rampup = getPropValue("rampup", defaultRampup).toLong()
         this.outputEnabled = getPropValue("outputEnabled", "true").toBoolean()
         this.testName = testName
     }
+
+    FloRunnaSettings(
+        String testName,
+        int threads,
+        int iterations,
+        long rampup
```

Comment:
Note: since the original signature method I had would match ones above, I just swapped `testName` to be first, to distinguish it from the others.